source/schema-configuration.txt

```diff
@@ -103,6 +123,28 @@ to update the schema.
 
 .. include:: /includes/fact-resample-schema-data.rst
 
+User Permissions
+~~~~~~~~~~~~~~~~
+
+To run |bi-short| in Persisted Schema mode, a MongoDB user needs 
+the :authrole:`readAnyDatabase` role and the :authrole:`readWrite` 
+role on the database that stores the schema (set using 
+:option:`--sampleMode`):
+
+.. code-block:: javascript
+   :class: copyable
+
+   db.getSiblingDB("admin").createUser(
+     {
+       user: "<username>",
+       pwd: "<password>",
+       roles: [
+          "readAnyDatabase",
+          { authrole: "readWrite", db: "<schemaDb>" }
+       ]
+     }
+   )
+
 .. _schema-with-drdl-file:
```

Comment:
A couple of thoughts.

Technically any of the all-database roles will do as far as permissions go. The simplest thing is to just instruct users to use the readAnyDatabase role, but any of them will work (https://docs.mongodb.com/manual/core/security-built-in-roles/#all-database-roles). I don't know if that's worth mentioning or not. In this passage https://docs.mongodb.com/bi-connector/current/reference/mongosqld/#security-considerations
I refer to any of the all-database roles.

Giving a code example showing how to create the correct kind of user is excellent. The example could maybe use a bit more context -- you could explain what the following command does, and that it must be run at the mongo shell prompt. Also you could mention that all this is necessary only if mongod and mongosqld are running with --auth turned on (as they always should be, but probably aren't).